### ISSUE TITLE
PaperUI incorrectly handling config type

### ISSUE THREAD

**Issue by cdjackson** [eclipse/smarthome#1441]
In ZWave I have a config description that looks like this -:

```
			<parameter name="heal_time" type="integer" groupName="heal">
				<label>Heal Time</label>
				<description></description>
				<default>2</default>
                <options>
                    <option value="-1">Disabled</option>
                    <option value="1">1 AM</option>
                    <option value="2">2 AM</option>
                    <option value="3">3 AM</option>
```

Updating the thing with this configuration throws a validation error that the type is integer and the data isn't (or something similar).

This works in HABmin.

**Comment by sjka** [comment]
Although your original point remains (i.e. the one about the PaperUI incorrectly sending strings instead of numbers), I think #1423 solved that issue since it's converting the Strings into BigIntegers during normalization.

**Comment by kaikreuzer** [comment]
@cdjackson We cannot reproduce your problem, the integer handling in the Paper UI seems to work as expected. We only discovered a but with 0 values, which is fixed by #1457.
If your problem remains, please provide some logs and re-open this issue - thanks!

**Comment by cdjackson** [comment]
Seems ok now - I was running a runtime that was a good few weeks old, but the latest from yesterday seems fine.
